ci/playbooks/build_runner_image.yml

```diff
@@ -42,8 +42,5 @@
       ansible.builtin.copy:
         mode: "0644"
         content: |
-          cifmw_edpm_prepare_update_os_containers: true
-          cifmw_set_openstack_containers_operator_name: openstack-ansibleee
-          cifmw_set_openstack_containers_overrides:
-            RELATED_IMAGE_ANSIBLEEE_IMAGE_URL_DEFAULT: "{{ ansibleee_runner_img }}"
+          cifmw_update_containers_ansibleee_image_url: "{{ ansibleee_runner_img }}"
```

Comment:
can this change break something relying on the old format for `edpm-ansible.yml`? In adoption for example we'll need a small patch to adapt to this change (not a problem, it'll be a simple change), but do we parse and use this file somewhere else?

Reply:
@cescgina Thank you for pointing that out. I was not aware about the dpa usage. Thank you for patching the dpa job. Based on my github search https://github.com/search?q=org%3Aopenstack-k8s-operators%20RELATED_IMAGE_ANSIBLEEE_IMAGE_URL_DEFAULT&type=code It is not used anywhere. We are passing edpm-ansible.yml as a extra var to the run playbook. https://github.com/search?q=org%3Aopenstack-k8s-operators+edpm-ansible.yml&type=code